solutions/cpython-tests/Makefile

```diff
@@ -109,6 +109,14 @@ To enable python source listing, do \n\
 before launching gdb.\033[0m\n')" \
            --args $(MYST_EXEC) $(OPTS) $(FS) /cpython/python -m unittest Lib.test.$(TESTCASE) -v
 
+testcase-mpdb:
+	killall myst 2> /dev/null || echo ""
```

Comment:
Is this related?

Reply:
This was useful in debugging cpython stacktraces. The related test `test_multiprocessing_spawn` is not part of this PR, being blocked by shared memory support.